src/navsat_transform.cpp

```diff
@@ -754,9 +754,9 @@ void NavSatTransform::setTransformGps(
   utm_meridian_convergence_ *= navsat_conversions::RADIANS_PER_DEGREE;
 
   RCLCPP_INFO(
-    this->get_logger(), "Datum (latitude, longitude, altitude) is (%d, %d, %d)",
+    this->get_logger(), "Datum (latitude, longitude, altitude) is (%f, %f, %f)",
```

Comment:
Can you limit the decimals please (`%0.2f`)

Reply:
Yep, thanks. Done.